NuGet.config

```diff
@@ -1,6 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
+    <clear />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
Why might I want to consider doing this?

Reply:
It's in the commit message https://github.com/DotNetAnalyzers/StyleCopAnalyzers/pull/3072/commits/a5cbd747cdc7e656f6d1815bbdcb7098b1e0123a

Reply:
Perfect, looked there but didn't think of looking for a multiline message. Thanks!